README.md

```diff
@@ -96,20 +91,20 @@ with a prebuilt set of phoenix crates. This is currently done by
 We can still use `cargo`.
 
 ### Running mRPC examples
-Once `rpc_hello` is built, you can have two methods to start it.
+<!-- Once `rpc_hello` is built, you can have two methods to start it.
 1. (Recommended) To start the applications on multiple machines, we prepare
 a launcher for this job.
 ```bash
 $ cd ../../benchmark
 Follow the README under benchmark directory and update config.toml
 $ cargo rr --bin launcher -- --benchmark benchmark/rpc_hello.toml
-```
+``` -->
 
 2. Alternatively, you can run the examples manually by
```

Comment:
Remove this `2. Alternatively` and maybe add a pointer to the [`launcher`](https://github.com/phoenix-dataplane/phoenix/tree/main/benchmark) at the end of this section.